### ISSUE TITLE
Create a process for project planning and execution

### ISSUE THREAD

**Issue by choldgraf** [2i2c-org/team-compass#205]
### Description

Create a role that is tasked with overseeing the refinement of our [team backlog](https://github.com/orgs/2i2c-org/projects/22/views/1), actively gathering input from stakeholders, and sharing context and information with the engineering team.

- [**Please provide feedback, commends, and edits in this proposal**](https://docs.google.com/document/d/11dM4aIzuTiXhC-U_v7uSTX6KUEDbKrNAb4JtNhpb_uk/edit?usp=sharing)

### Tasks to complete

- [x] Write up a draft of the backlog shaping role / process
- [x] Feedback and comments on the draft above
- [x] https://github.com/2i2c-org/team-compass/issues/321
- [ ] Take an incremental step towards this proposal (see https://github.com/2i2c-org/team-compass/issues/205#issuecomment-988210590)
- [ ] Re-assess in February 2022
- [ ] Merge into the team compass

### Updates

- **2021-12-07** - [Decided to take baby steps in this direction and re-assess later](https://github.com/2i2c-org/team-compass/issues/205#issuecomment-988210590)

**Comment by choldgraf** [comment]
### Update: Team meeting discussion

In a recent team meeting, we discussed the idea of having this be a **role** vs. a dedicated person. We didn't make any strong decisions, but the general feeling was that "product management" or "backlog management" may be a complex-enough role that it warrants dedicated time from people that specialize in this, rather than cycling throughout the whole team.

**Comment by choldgraf** [comment]
### Update: a proposal

I've created a proposal with three ways that I think we can improve our project planning and coordination process. Briefly, this includes:

- Define a team role for "Backlog Steward", who oversees our backlog design and shaping process, and who gathers input from stakeholders and shares context with the development team.
- Define a weekly "backlog refinement" meeting. This is **optional** for team members, but all are welcome to join. In this meeting we'll go over the backlog, discuss major items, and try to get them to a ready-state.
- Sharpen our "Projects" to be scoped more tightly, shorter in time, and with clear deliverables outcomes. Team members will dedicate their time to these projects through to their completion.
-

**Comment by choldgraf** [comment]
### Update: More additions to the proposal

After several conversations with other team members, and a few more iterations, the proposal is updated with a slightly different process above. Let's discuss these ideas in our team meeting tomorrow! https://docs.google.com/document/d/11dM4aIzuTiXhC-U_v7uSTX6KUEDbKrNAb4JtNhpb_uk/edit#

**Comment by choldgraf** [comment]
### Update: baby steps

We discussed this topic in the meeting today, and had a quick once-over of the proposal. In general people seemed to think the steps would help, but there was concern that adopting everything at once would be too disruptive to our team practices.

Instead, we agreed that we should take some baby steps in this direction and make incremental improvements, here's a rough idea of some things we can do now:

**Team Backlog**
- Remove most development deliverables, replace with longer-term projects that are well-shaped.
- Projects have an appetite associated with them, and we commit to working on that project only for its appetite.
- Split the operations items into a separate category that we track in parallel

**Sprint meeting**
- People pick up projects, rather than deliverables. assign in pairs so that there's a champion and an assistant.
- If someone already has a project, they remain on it.
- New operational items may be assigned to team members, but not new projects.

**New (optional) meeting: Project Shaping**
- Chris holds a weekly meeting to discuss project pitches and shape them. This is optional for anybody else to attend, others are encouraged to come and discuss ideas, or to pitch their own new ideas.
- In the future, this might be led by a "Product Lead" role

**Comment by choldgraf** [comment]
### Update: splitting this into multiple issues

I had a meeting with @damianavila last week and we agreed that we should split this large issue / proposal into a few more discrete ones that we can start to experiment with. So I've created the following issues, and will close this one:

- https://github.com/2i2c-org/team-compass/issues/443
- https://github.com/2i2c-org/team-compass/issues/444
- https://github.com/2i2c-org/team-compass/issues/445
